src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -263,6 +267,12 @@ class DocumentsSpec extends CouchDbSpecification {
         documents.getMany.includeDocs[FixXPerson].byTypeSimple(FixViews.typeFilter, fixDesign.name,
```

Comment:
Sorry, I missed this earlier, but is it even worth fixing the value type? I guess setting the key and type to strings by defaults makes sense as the common case, but are string values common enough case? Also, `byTypeSimple` is not a very descriptive name, hard to tell from the name what it does. Another small issue I noticed is the deprecation messages that have `byType[K, V, D]` need to be updated to remove `D`.